src/Native/CMakeLists.txt

```diff
@@ -86,6 +86,11 @@ if(CMAKE_SYSTEM_NAME STREQUAL NetBSD)
     set(CLR_CMAKE_PLATFORM_NETBSD 1)
 endif(CMAKE_SYSTEM_NAME STREQUAL NetBSD)
 
+if(CMAKE_SYSTEM_NAME STREQUAL Emscripten)
+    set(CLR_CMAKE_PLATFORM_UNIX 1)
+    set(CLR_CMAKE_PLATFORM_UNIX_TARGET_WASM 1)
```

Comment:
WebAssembly isn't really UNIX, would it make sense for this variable to just be called CLR_CMAKE_PLATFORM_WASM ?